recipe/meta.yaml

```diff
@@ -10,17 +10,16 @@ source:
   sha256: 6a769e4816921ac4059c8c149ab5f69ed7cd92859857f0e17b67a3dd7bbee866
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
-    - python >=3.8
+    - python
     - hatchling
     - pip
   run:
-    - python >=3.8
+    - python
```

Comment:
there are some run_constraineds you could add in ([upstream](https://github.com/secure-systems-lab/securesystemslib/blob/fe0cf39961936d7e66d0fc0d104371dfd441f1da/pyproject.toml#L45); [main branch of this recipe](https://github.com/AnacondaRecipes/securesystemslib-feedstock/blob/1bd0d4686a2015cebd6a5ebb7198bef7fb315f52/recipe/meta.yaml#L23))

Reply:
+1

Reply:
I agree but I also wonder if we should add upper bounds as well since this is an older version.